package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
THe alternative is to change the label to `vscode-icons` as well and dispute gets solved.